### PR TITLE
Add a Crash Handling subsection.

### PR DESCRIPTION
**Pull request by al45tair**
Add some words about the new crash handler, including a promise of a future blog post on the subject.

### CODE REVIEW

**Review by alexandersandberg — APPROVED** [review]
Thanks! 👍

File: _posts/2023-08-29-swift-5.9-released.md [hunk 1]
```diff
@@ -82,6 +82,14 @@ A full list of Swift 5.9's Evolution proposals can be found at the end of this p
 
 <!-- TODO: Add more sections with content? -->
 
+### Crash Handling
+
+On Linux, the Swift runtime will now catch program crashes and Swift runtime errors and display a backtrace on the program's output.  The backtracer is out-of-process and includes support for `async` functions.
+
+This feature is also available on macOS, although there it is disabled by default (it can be enabled by setting `SWIFT_BACKTRACE=enable=yes`) and you need to ensure that your program is signed with the `com.apple.security.get-task-allow` entitlement.
+
+A more detailed blog covering this feature will be published soon.
```
Comment (tomerd) [inline _posts/2023-08-29-swift-5.9-released.md:91]: "soon" is a relative term. maybe we can reword this so it is evergreen? alternatively we can drop this line

Comment (al45tair) [reply _posts/2023-08-29-swift-5.9-released.md:91]: The section immediately below this one said a blog post was coming on the debugger "next week". Given that I don't have any idea how the blog post approval process works or how long it takes, I couldn't very well say that, but I think we should do a post about it (I've written something already).

If "soon" is wrong, maybe "in due course"? Though that sounds rather formal?

Comment (tomerd) [reply _posts/2023-08-29-swift-5.9-released.md:91]: I think maybe we just drop this line.

Comment (al45tair) [reply _posts/2023-08-29-swift-5.9-released.md:91]: OK. I'll raise another PR, since this one got merged.

**Pull request merged**